Specs/TypeScript/index.ts

```diff
@@ -379,3 +379,7 @@ geometryInstance = new GeometryInstance({
     positions: [],
   }),
 });
+
+let pb = new PropertyBag();
+let hasProp: boolean = pb.hasProperty("xyz");
+property = pb.xyz;
```

Comment:
This assignability check passes but only because we're currently stuck with `any` as the index operator return type.  Once Negated Types are introduced it should be possible to have these dynamic properties typed correctly as `Property`.